src/connectors/vocadb.ts

```diff
@@ -0,0 +1,54 @@
+export {};
+
+const filter = MetadataFilter.createFilter({
+	artist: cleanupArtist,
+});
+
+Connector.applyFilter(filter);
+
+function cleanupArtist(artist: string) {
+	switch (location.hostname) {
+		case 'vocadb.net':
+			return artist.replace(/(?<=feat. ).+$/, function(match: string) {
+				let feat = match.split(', ');
+				for (let i = 0; i < feat.length; i++) {
+					feat[i] = feat[i]
+						.replace(/\s+\([^()]+\)/g, '')
+						.replace(/(\s(V\d+X?|Append|English|AI|NT|Sugar|Spicy|ROCKS|2S|Standard|II|TALK)\b.*)+$/gi, '')
+						.replace(/\s(ナチュラル|クール)$/gi, '')
+						.replace(/^(AI|V\d+X?)+\s/gi, '')
+						.replace(/^(VY\d+)V\d+\b/, '$1')
+						.replace(/^結月ゆかり.+$/, '結月ゆかり')
+						.replace(/^波音リツキレ$/, '波音リツ')
+						.replace(/^星尘Infinity$/, '星尘')
+						.replace(/^重音テトSV$/, '重音テト')
+						.replace(/^遙$/, '夏語遙')
+						.replace(/^v flower|v4 flower|Ci flower$/, 'flower')
+						.trim();
+				}
+				return [...new Set(feat)].join(', ');
+			});
+			break;
+		case 'utaitedb.net':
+			return artist.replace(/^.+ feat\. /, '');
+			break;
+		default:
+			return artist;
+	}
+}
+
+Connector.playerSelector = '#app .css-1pm1wrk';
```

Comment:
These selectors still feel quite brittle to any new deployment as mentioned in https://github.com/web-scrobbler/web-scrobbler/pull/3596#discussion_r1142775713

Is there any other way the selection could be done?

Reply:
@sorusari343 could you change to use the selectors noted above and fix the linting issues? Or would you like me to help with that?

Reply:
Sorry, I didn't see these replies earlier. I think the website has been through a few software updates and the selectors seem to still work, and as such, I did not give that part much thought. I turned edits by maintainers on, so please do whatever is necessary.